src/pages/index.js

```diff
@@ -19,9 +85,55 @@ const IndexPage = () => {
         />
       </Helmet>
 
-      <h1> Raunaq, time to do shit :) </h1>
+      <Navbar
+        sticky="top"
+        expand="lg"
+        style={{
+          backgroundColor: '#FFFFFF',
+          boxShadow: '0px 5px 6px #00000029',
+          fontSize: '0.9rem',
+          flexDirection: 'row',
+        }}
+      >
+        <img src={require('../../static/headerlogo.png')} height="50" />
```

Comment:
check this out https://www.gatsbyjs.com/docs/recipes/working-with-images/#reference-an-image-from-the-static-folder